src/libs/dataUseTranslation.js

```diff
@@ -10,6 +104,86 @@ export const DataUseTranslation = {
    * @param darInfo
    * @returns {{primary: [{code: '', description: ''}], secondary: [{code: '', description: ''}]}}
    */
+
+  //NOTE: backend categorization of purposeStatement/researchType differs from front-end primary/secondary designations
+  //Should the two categories be unified or are they distinct for a reason?
```

Comment:
The back end categorization of `purposeStatement/researchType` is out of date which is why we want to move them to the front end and make them consistent with how we're describing primary/secondary designations.